api/opencl_architecture.asciidoc

```diff
@@ -1660,7 +1660,7 @@ all_svm_devices scope.
 
 
 [[memory-ordering-fence]]
-==== Memory Ordering Rules: Fence Operations
+==== Fence Operations
```

Comment:
Can you update the link text on Line 1139 so it is consistent with this change?

Reply:
Yes, fixed that now.  Thanks for the review.